src/main/java/com/dev/cinema/Main.java

```diff
@@ -46,5 +51,26 @@ public static void main(String[] args) throws AuthenticationException {
         System.out.println(bobRegister);
         User bobLogin = authenticationService.login("bob@gmail.com", "bobpass");
         System.out.println(bobLogin);
+
+        Ticket ticket1 = new Ticket();
+        ticket1.setUser(bobLogin);
+        ticket1.setMovieSession(movieSession);
+        Ticket ticket2 = new Ticket();
+        ticket2.setUser(bobLogin);
+        ticket2.setMovieSession(movieSession);
+        List<Ticket> tickets = new ArrayList<>();
+        tickets.add(ticket1);
+        tickets.add(ticket2);
```

Comment:
let's remove this and use `shoppingCartService.addSession()` to add ticket to cart